xclim/data/fr.json

```
@@ -912,6 +912,42 @@
     "title": "Date du fin du couvert de neige",
     "abstract": "Première date à partir de laquelle l'épaisseur de neige est sous à un seuil donné pendant un nombre de jours consécutifs suite au début du couvert de neige."
   },
+  "SNOW_DEPTH_COVER_DURATION": {
+    "long_name": "Nombre de jour où l'épaisseur de neige est au-dessus d'un seuil",
+    "description": "Nombre {freq:m} de jours où l'épaisseur de neige est au-dessus ou égale à {thresh}.",
+    "title": "Durée du couvert de neige",
+    "abstract": "Nombre de jours pendant lesquels l'épaisseur de neige est au-dessus ou égale à un seuil donné."
+  },
+  "CONTINUOUS_SNOW_DEPTH_COVER_START": {
+    "long_name": "Date du début du couvert de neige continu",
+    "description": "Première date à laquelle l'épaisseur de neige est au-dessus ou égale à {thresh} pendant au moins {window} jours consécutifs.",
+    "title": "Date du début du couvert de neige",
+    "abstract": "Première date à partir de laquelle l'épaisseur de neige est au-dessus ou égale à un seuil donné pendant un nombre de jours consécutifs."
+  },
+  "CONTINUOUS_SNOW_DEPTH_COVER_END": {
+    "long_name": "Date de fin du couvert de neige continu",
+    "description": "Première date à laquelle l'épaisseur de neige passe sous {thresh} pendant au moins {window} jours consécutifs suite à l'établissement du couvert de neige.",
+    "title": "Date du fin du couvert de neige",
+    "abstract": "Première date à partir de laquelle l'épaisseur de neige est sous à un seuil donné pendant un nombre de jours consécutifs suite au début du couvert de neige."
+  },
+  "SNOW_AMOUNT_COVER_DURATION": {
+    "long_name": "Nombre de jour où la quantité de neige est au-dessus d'un seuil",
+    "description": "Nombre {freq:m} de jours où la quantité de neige est au-dessus ou égale à {thresh}.",
+    "title": "Durée du couvert de neige",
```

Same as the other above.
```suggestion
    "title": "Durée du couvert de neige (quantité)",
```